reader-osm/src/test/java/com/graphhopper/reader/osm/GraphHopperOSMTest.java

```diff
@@ -887,7 +888,7 @@ public void testGetPathsDirectionEnforcement6() {
 
     private GraphHopper createSquareGraphInstance() {
         CarFlagEncoder carEncoder = new CarFlagEncoder();
-        EncodingManager encodingManager = EncodingManager.create(carEncoder);
+        EncodingManager encodingManager = new EncodingManager.Builder().add(carEncoder).add(InSubnetwork.create("profile")).build();
```

Comment:
Hm this is another case of this strange `loadGraph` method and really these tests have nothing to do with subnetwork removal...

Reply:
Fixed in #2291